common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -406,7 +406,7 @@ div.problem {
       margin-top: 3px;
 
       .MathJax_Display {
-        display: inline-block;
+        display: inline-block !important;
```

Comment:
I believe this rule is used in textlines with math display and with formulaequation inputs only. Regardless, as this rule is being overwritten by the !important flag anyway, I've gone ahead and removed it, and added a specific case for the formulaequationinput.